samples/sensor/max31865/src/main.c

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2022 HAW Hamburg FTZ-DIWIP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr/zephyr.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/sys/printk.h>
+
+union sensor_val {
+	struct sensor_value str;
+	uint64_t wert;
+
+};
+
+void main(void)
+{
+	k_msleep(1000);
+
+	const struct device *max_sensor = DEVICE_DT_GET_ANY(maxim_max31865);
+	union sensor_val sv;
+	int r;
+
+	while (1) {
+
+		r = sensor_sample_fetch(max_sensor);
+		if (r != 0) {
+			continue;
+		}
+		sensor_channel_get(max_sensor, SENSOR_CHAN_AMBIENT_TEMP, &sv.str);
```

Comment:
let's not add yet another sensor sample without being generic for the sensor type (unless it's really needed to demonstrate special features). See the work done for the generic accelerometer polling sample.

Reply:
So should the sample be removed?